### PR TITLE
remove v1.19-v1.22 generated api ref

### PR DESCRIPTION
**Pull request by kbhawkey**
- removed generated api references v1.19-v1.22

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 
 ignoreFiles = [ "(?:^|/)OWNERS$", "README[-]+[a-z]*\\.md", "^node_modules$", "content/en/docs/doc-contributor-tools" ]
 
-timeout = "180s"
+timeout = "100s"
```
Comment (tengqm) [inline hugo.toml:18]: This can be in a separate PR?

Comment (kbhawkey) [reply hugo.toml]: I removed this change.

**Comment by sftim** [comment]
Seems reasonable given we don't support older docs versions (something to revisit when we have capacity).

/lgtm
